.github/workflows/End2EndTest.yml

```diff
@@ -66,7 +66,69 @@ jobs:
           ./scripts/decrypt_secret_windows.ps1 -SnowflakeDeployment '${{ matrix.snowflake_cloud }}'
       - name: Unit & Integration Test (Windows)
         continue-on-error: false
-        run: mvn -DghActionsIT verify --batch-mode
+        run: |
+          mvn -DghActionsIT -D"failsafe.excludedGroups"="net.snowflake.ingest.IcebergIT" verify --batch-mode
+  build-iceberg:
```

Comment:
Let's add a comment about why this is needed, as well as a TODO with this is temporary and can be consolidated in the future